lua/ibl/utils.lua

```diff
@@ -231,4 +246,139 @@ M.tbl_get_index = function(list, i)
     return list[((i - 1) % #list) + 1]
 end
 
+---@param bufnr number
+---@param highlight string|string[]
+---@param start_row number
+---@param start_col number
+---@param end_row number
+---@param end_col number
+---@param fallback_index number
+---@return number
+M.highlight_from_extmark = function(bufnr, highlight, start_row, start_col, end_row, end_col, fallback_index)
+    if type(highlight) ~= "table" or start_row == math.huge then
+        return fallback_index
+    end
+
+    local start_line = vim.api.nvim_buf_get_lines(bufnr, start_row, start_row + 1, false)
+    local end_line = vim.api.nvim_buf_get_lines(bufnr, end_row, end_row + 1, false)
+    local end_pos
+    local start_pos
+    local start_pos_col
+    local end_pos_col
+
+    if end_line[1] then
+        end_pos = vim.inspect_pos(bufnr, end_row, end_line[1]:find "%S" - 1, {
```

Comment:
```suggestion
        local end_pos_find = #end_line[1] - 1
        local try_find = end_line[1]:find "%S"
        if try_find then
            end_pos_find = try_find - 1
        end
        end_pos = vim.inspect_pos(bufnr, end_row, end_pos_find, {
```
The current code courses problems when find returns `nil` sometimes.